### PR TITLE
docs: site/how-to clean up

### PR DESCRIPTION
**Pull request by lunamidori5**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by mudler — APPROVED** [review]

**Comment by lunamidori5** [comment]
Added do not merge, I am going to get the site updated for 2.0 then push this PR

**Comment by lunamidori5** [comment]
@mudler its ready for merge, sorry for confusing post, there is a fix due to a few things wrong with the autogen pages and stuff this also fixes

File: docs/content/faq/_index.en.md [hunk 1]
```diff
@@ -96,3 +96,11 @@ This typically happens when your prompt exceeds the context size. Try to reduce
 Your CPU probably does not have support for certain instructions that are compiled by default in the pre-built binaries. If you are running in a container, try setting `REBUILD=true` and disable the CPU instructions that are not compatible with your CPU. For instance: `CMAKE_ARGS="-DLLAMA_F16C=OFF -DLLAMA_AVX512=OFF -DLLAMA_AVX2=OFF -DLLAMA_FMA=OFF" make build`
   
 </details>
+
+### I'm getting a 'SIGILL' error, what's wrong?
+
+<details>
```
[the remark this answers is not in the document]
Comment (lunamidori5) [reply docs/content/faq/_index.en.md:102]: That shouldn't be there. I didn't even touch that part I don't think.

**Comment by lunamidori5** [comment]
@mudler fixed sorry about that

**Review by mudler — APPROVED** [review]

**Pull request merged**